doc/md/features.mdx

```diff
@@ -48,49 +48,50 @@ Schema Visualization as well as Interactive Migrations are features that require
 
 All common open-source RDBMS are supported in all versions of Atlas. In addition, Atlas also supports SQL Server and ClickHouse in the Free Plan of Atlas Cloud.
 
-| Driver       | Availability     |
-|--------------|------------------|
-| MySQL        | Open |
-| PostgreSQL   | Open |
-| MariaDB      | Open |
-| SQLite       | Open |
-| LibSQL       | Open |
-| SQL Server   | Requires Login (Beta) |
-| ClickHouse   | Requires Login (Beta) |
+| Driver     | Availability          |
+|------------|-----------------------|
+| MySQL      | Open                  |
+| PostgreSQL | Open                  |
+| MariaDB    | Open                  |
+| SQLite     | Open                  |
+| LibSQL     | Open                  |
+| SQL Server | Requires Login        |
+| ClickHouse | Requires Login (Beta) |
 
 ### Database Features
 
 Most common database features are supported in all versions of Atlas. More advanced features are available by running `atlas login`.
 
-| Feature                  | Availability     |
-|--------------------------|------------------|
-| Tables and Columns       | Open |
-| Indexes                  | Open |
-| Foreign Keys             | Open |
-| Constraints              | Open |
-| Partitions               | Open |
-| Views                    | Requires Login   |
-| Materialized Views       | Requires Login   |
-| Stored Procedures        | Requires Login   |
-| Functions                | Requires Login   |
-| Triggers                 | Requires Login   |
-| Sequences (Postgres)     | Requires Login   |
-| Domain Types (Postgres)  | Requires Login   |
+| Feature                 | Availability   |
+|-------------------------|----------------|
+| Tables and Columns      | Open           |
+| Indexes                 | Open           |
+| Foreign Keys            | Open           |
+| Constraints             | Open           |
+| Partitions              | Open           |
+| Views                   | Requires Login |
+| Materialized Views      | Requires Login |
+| Stored Procedures       | Requires Login |
+| Functions               | Requires Login |
+| Triggers                | Requires Login |
+| Sequences (Postgres)    | Requires Login |
+| Domain Types (Postgres) | Requires Login |
```

Comment:
May be change this to `User-Defined Types`

Reply:
I can add another one for udf/composite types after it will be added. The entire sqlserver driver requires login, so this one is for the open drivers.